src/__tests__/components/Image.test.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
```

Comment:
Glad you added unit testing! 👌🏽

Reply:
Have you tried the React Testing Library?

Reply:
I have worked with React Testing Library but the thing here was I had some troubles using it with Codesanbox (Never used before). I know there are more test cases I could add, I was trying to create a real snapshot test for components but Codesanbox did not let me create the first snapshot file so I went to this simpler solution.